2024/2024-ssl-teams-vs-teams-passing-challenge.adoc

```diff
@@ -0,0 +1,105 @@
+
+== Procedure
+
+The technicall challenge will be held on a Div B field with a maximum of 4 robots per team.
+
+To level the playing field and to remove the impact of the performance of the opponent team on a teams score, each team will play two games against different contestants of the technical challenge.
+
+Each game will have the following procedure:
+
+* If both teams agree on the team colors, their suggestion will be used. If they do not agree, the referee will make a coin toss and the winning team will get the yellow team color. Even in this case, the teamcolor will not be changed during the half time break.
+* To start the game, a kick-off command for the yellow team will be issued from the game controller. For the start the robots can be positioned can positioned in any way prefered, along as the positioning does not violate the usual ruleset for a kick-off.
+* After *2 minutes of game time* a stop command will be issued and the first half of the game is over.
+* The half time break for this challenge will be only 60 seconds, since it should be mainly used to setup the robots at their desired position again and to exchange faulty robots. 
+* To start the second half a kick-off command for the blue team will be issued.
+* After another *2 minutes of game time* a stop command will be issued and the game is over. 
+
+
+=== Pass Definitions
+
+Teams are free in how they perform the passes. Definitions of a pass in general and additional constrains for valid pass that are considered in the scoring are given in this section.
+
+A pass is defined as follows:
+
+* A pass has a source position, a target position and a shooter.
+* The source position is where a robot kicks the ball.
+* The target position is where the ball stops or changes direction by more than *30 degrees* in field plane (x, y).
+* The shooter is the robot located nearest to the source position when the pass starts.
+* The pass distance is the euclidean distance between the source and target position.
+* The pass direction is the orientation of the `target - source` vector.
+* The pass is considered *chipped*, if the ball reached a maximum height of at least *15cm*.
+
+=== Special Ruleset during the technical challenge
+
+For the technical challenge a Div B field will be used. If the ball leaves the field via any of the field lines, the game will be halted and the team, that did not last touch the ball gets awared a free kick.
+
+During the games of the technical challenge only the following fouls from the ssl rulebook apply: 
+
+* Pushing 
+* Ball Holding 
+* Tipping over or dropping parts 
+* Boundary Crossing 
+* Excessive Dribbling 
+* Ball Speed 
+* Crashing 
+* Defender too close to ball 
+* Robot Stop Speed 
+
+During this challenge a maximum of 4 robots per team are allowed. For every yellow card, that a team got, the number of allowed robots for the half time will be decreased by one. A red card will decrease that number for the entire time of the game. 
```

Comment:
Really only for the half time? It would be easier to transfer it to the second half, otherwise we need to manually remove the cards in the GC during half time. Also, the punishment would be minimal, when a foul is performed shortly before the end of the half time.

Reply:
I removed the section, so the yellow cards will keep their normal behavior.